src/win/pty.cc

```diff
@@ -131,6 +131,23 @@ void throw_winpty_error(const char *generalMsg, winpty_error_ptr_t error_ptr) {
   winpty_error_free(error_ptr);
 }
 
+static NAN_METHOD(PtyGetExitCode) {
+  Nan::HandleScope scope;
+
+  if (info.Length() != 1
+    || !info[0]->IsNumber()) // pidHandle
+  {
+    Nan::ThrowError("Usage: pty.getExitCode(pidHandle)");
+    return;
+  }
+
+  uint32_t pidHandle = info[0]->Uint32Value();
+  DWORD exitCode = 0;
+  GetExitCodeProcess((HANDLE)pidHandle, &exitCode);
```

Comment:
@Tyriar Did some tests, seems I cant get Windows to use very high process handles, they kinda always stick in 10.000s (I did not want to overflow my memory with millions of processes). Imho either way, signed or unsigned, should be safe for a legal process handle. Only trap I can think of is the `INVALID_HANDLE_VALUE` which is defined as -1 and would turn into a legal handle by using `uint32_t` instead of `int32_t`.